### DISCUSSION TITLE
Import third party pcss file?

### DISCUSSION THREAD

**Discussion by SpiderRam** [vercel/next.js#33320] [Help]
I have a pcss stylesheet, published in npm.  I have different classes that allow me to change the look of an app by changing a property to set the class on a component.   For example:

```pcss
:root {
  --base-color: #000;
}

.earth {
  --base-color: #10A504;
)

.ocean {
  --base-color: #0076FF;
}
```

I have created a next.config.js file:

```javascript
module.exports = {
  webpack: (config, { buildId, dev, isServer, defaultLoaders, webpack }) => {
    config.module.rules.push({
        test: /\.css$/,
        exclude: /node_modules/,
        use: [
          {
            loader: 'style-loader',
          },
          {
            loader: 'css-loader',
            options: {
              importLoaders: 1,
            }
          },
          {
            loader: 'postcss-loader'
          }
        ]
      })
    return config
  },
}

```

And an _app.js file:
```javascript
import 'mycss/src/styles/colors.pcss'
import './index'

export default function MyApp({ Component, pageProps }) {
  return <Component {...pageProps} />
}
```

I am exploring new frameworks and part of the acceptance criteria is to be able to use this stylesheet as-is.

The code being pushed in above is from the [postcss docs](https://github.com/postcss/postcss#webpack).

I am getting an error  "You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file."  I have the loaders installed, and a postcss.config.js file.

It is choking on the first line from the stylesheet:

`:root {
    --base-color: #000;
 }`
 
 Any insight into whether this is possible/recommended with next.js, or any obvious configuration detail I might be overlooking, is welcome.

**Comment by danielweil** [comment]
I also need help with this same exact issue

**Comment by SpiderRam** [comment]
Ah - actually, it looks like this is the built-in behavior, and not something I can get past.  Looks like Next is not right for me, but leaving this [link](https://github.com/vercel/next.js/issues/19936) here for anyone else.
